.github/workflows/ci.yml

```diff
@@ -63,6 +63,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install prerequisites
         run: |
+          pip3 install -U Jinja2
```

Comment:
Have you searched for issues like this? Don't we expect any fix on third-party side?

Reply:
Upstream issue: https://github.com/pallets/jinja/issues/870 (Fixed in version 2.11.0)
Ubuntu-20.04 has jinja2 2.10.1